how-tos/maintain/upgrade-notes-sql-mongodb/upgrade-to-v4.md

```diff
@@ -37,6 +37,10 @@ npm install forest-express-mongoose@4.1.2
 In case of a regression introduced in Production after the upgrade, a rollback to your previous agent version 3 is the fastest way to restore your admin panel.
 {% endhint %}
 
+{% hint style="info" %}
```

Comment:
I would move it at the beginning, line 15 I would say.
Just after the breaking changes warning.